src/ChunkedEncoderStream.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace Legionth\React\Http;
+
+use Evenement\EventEmitter;
+use React\Stream\ReadableStreamInterface;
+use React\Stream\WritableStreamInterface;
+use React\Stream\Util;
+
+/**
+ * Stream incoming strings as a HTTP chunked encoded string
+ */
+class ChunkedEncoderStream extends EventEmitter implements ReadableStreamInterface
```

Comment:
What "incoming strings"? Perhaps something like "Wraps any readable stream which may emit number of data chunks as strings", also some documentation about the *why*?